content/projects/nodejs/file-io/_index.md

```diff
@@ -84,4 +84,17 @@ Your load function should also get a bit of an update.
 charlie = load(3)
 charlie.comments = "Winning!"
 charlie.save() # results in an UPDATE to visitor_3.json
-```
\ No newline at end of file
+```
+
+## Instructions for reviewer
+
+
+- Please make sure that the save method saves the data in lower case and that whitespace is replaced by an underscore. This is the only accepted format.
+
+- The use of string literals is highly encouraged for the save method. It's good if a recruit uses them
```

Comment:
This seems more of a tip for the person doing the project than it is for the reviewer.